sympy/polys/domains/gmpyrationalfield.py

```diff
@@ -11,10 +11,10 @@
 
 @public
 class GMPYRationalField(RationalField):
-    """Rational field based on GMPY's ````mpq```` type.
+    """Rational field based on GMPY's ``mpq`` type.
 
-    This will be the implementation of :ref:``QQ`` if ````gmpy```` or ````gmpy2```` is
-    installed. Elements will be of type ````gmpy.mpq````.
+    This will be the implementation of :ref:``QQ`` if ``gmpy`` or ``gmpy2`` is
+    installed. Elements will be of type ``gmpy.mpq``.
```

Comment:
Does this docstring actually show up in the docs?

I think that the ref needs to be
```
:ref:`QQ`
```
but I'm not sure.